class.jetpack-jitm.php

```diff
@@ -256,6 +256,7 @@ function get_messages( $message_path, $query ) {
 		$path = add_query_arg( array(
 			'external_user_id' => urlencode_deep( $user->ID ),
 			'query_string'     => urlencode_deep( $query ),
+			'mobile_browser'   => jetpack_is_mobile('smart') ? 1 : 0,
```

Comment:
Yep, looks good. This also needs to be whitelisted in the api endpoint in wpcom.